app/controllers/vocabularies_controller.rb

```diff
@@ -1,5 +1,6 @@
 class VocabulariesController < ApplicationController
   before_filter :load_vocab, :only => :show
+  before_filter :authorize, :only => [:new, :create]
```

Comment:
Would it make more sense to provide an `:except` list rather than an `:only` list?  I'd think we'd want auth by default for when we add new functionality.